dynawo/sources/Models/Modelica/Dynawo/Electrical/Wind/IEC/WPP/WPP4CurrentSource_INIT.mo

```diff
@@ -0,0 +1,229 @@
+within Dynawo.Electrical.Wind.IEC.WPP;
```

Comment:
It looks like it's the same as WT4CurrentSource_INIT so please use only one model, or at least extend one, thanks.